tomcat/src/main/java/org/apache/catalina/servlet/ServletMapping.java

```diff
@@ -1,18 +1,18 @@
 package org.apache.catalina.servlet;
 
 import java.util.List;
-import nextstep.jwp.exception.NotSupportedRequestException;
 import nextstep.jwp.handler.DispatcherServlet;
+import org.apache.catalina.exception.NotSupportedRequestException;
 import org.apache.coyote.http.vo.HttpRequest;
 
 public class ServletMapping {
 
-    private static final List<HttpServlet> servlets = List.of(
+    private static final List<Controller> servlets = List.of(
             new DefaultServlet(),
             new DispatcherServlet()
     );
 
-    public static HttpServlet getSupportedServlet(final HttpRequest request) {
+    public static Controller getSupportedServlet(final HttpRequest request) {
         return servlets.stream().filter(it -> it.isSupported(request))
                 .findFirst()
                 .orElseThrow(NotSupportedRequestException::new);
```

Comment:
```suggestion
    public static Controller getSupportedServlet(final HttpRequest request) {
        return servlets.stream()
                .filter(it -> it.isSupported(request))
                .findFirst()
                .orElseThrow(NotSupportedRequestException::new);
```

ㅎㅎ 컨벤션 다 확인해보는 건 항상 힘든 것 같아요 저도 그렇고 ㅠㅠ
블랙캣의 더 완성도 있는 코드가 되길 바라며 제안해봅니다 👍